tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -524,7 +533,8 @@ def _append_plugin_asset_directories(self, run_path_pairs):
                 metadata.PLUGIN_ASSETS_NAME,
```

Comment:
nit: maybe replace this with `plugin_assets_name`, since we've extracted it in L525?

Reply:
Ah, indeed. Thanks.